File: Lehrverwaltung/server/src/main/java/tie/lehrlinge/lehrverwaltung/server/model/UserInfoModel.java
package tie.lehrlinge.lehrverwaltung.server.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
public class UserInfoModel {

  private String username;
  private String firstname;
  private String lastname;
  private String role;
}

File: Lehrverwaltung/client/src/app/core/models/userInfo/UserInfoModel.ts
export class UserInfoModel {
  username: string;
  firstname: string;
  lastname: string;
  role: string;

  constructor(username: string, firstname: string, lastname: string, role: string) {
    this.username = username;
    this.firstname = firstname;
    this.lastname = lastname;
    this.role = role;
  }
}
